src/test/e2e/api/admin/user-admin.e2e.test.ts

```diff
@@ -32,463 +31,536 @@ let sessionStore: ISessionStore;
 let editorRole: IRole;
 let adminRole: IRole;
 
-beforeAll(async () => {
-    db = await dbInit('user_admin_api_serial', getLogger);
-    stores = db.stores;
-    app = await setupAppWithCustomConfig(stores, {
-        experimental: {
-            flags: {
-                strictSchemaValidation: true,
-                showUserDeviceCount: true,
+describe('User Admin API with email configuration', () => {
+    beforeAll(async () => {
+        db = await dbInit('user_admin_api_serial', getLogger);
+        app = await setupAppWithCustomConfig(
+            db.stores,
+            {
+                email: {
+                    host: 'smtp.ethereal.email',
+                    smtpuser: 'rafaela.pouros@ethereal.email',
+                    smtppass: 'CuVPBSvUFBPuqXMFEe',
+                },
+                experimental: {
+                    flags: {
+                        strictSchemaValidation: true,
+                        showUserDeviceCount: true,
+                    },
+                },
             },
-        },
+            db.rawDatabase,
```

Comment:
This ensures we use the  transactional service instead of the fake transactional one